extensions/spark/kyuubi-spark-connector-tpcds/benchmarks/TPCDSTableGenerateBenchmark-results.txt

```diff
@@ -0,0 +1,34 @@
+Java HotSpot(TM) 64-Bit Server VM 1.8.0_311-b11 on Windows 10 10.0
+AMD64 Family 23 Model 96 Stepping 1, AuthenticAMD
+TPCDS table generates 1000000 rows benchmark:  Best Time(ms)   Avg Time(ms)   Stdev(ms)    Rate(M/s)   Per Row(ns)   Relative
+----------------------------------------------------------------------------------------------------------------------------
+catalog_returns benchmark                             13519          13685         166          0.1       13519.1       1.0X
+catalog_sales benchmark                               11968          12075         134          0.1       11968.2       1.1X
+customer benchmark                                     7319           7666         600          0.1        7319.3       1.8X
+customer_address benchmark                             4756           4814         132          0.2        4755.7       2.8X
+customer_demographics benchmark                         857            864           4          1.2         857.4      15.8X
+inventory benchmark                                     524            526           2          1.9         524.2      25.8X
+store_returns benchmark                               11568          11646          89          0.1       11567.6       1.2X
+store_sales benchmark                                  9371           9446          59          0.1        9370.8       1.4X
+web_returns benchmark                                 13335          13395          35          0.1       13334.8       1.0X
+web_sales benchmark                                   12491          12548          60          0.1       12491.5       1.1X
+
+Java HotSpot(TM) 64-Bit Server VM 1.8.0_311-b11 on Windows 10 10.0
+AMD64 Family 23 Model 96 Stepping 1, AuthenticAMD
+TPCDS table generates all rows benchmark:  Best Time(ms)   Avg Time(ms)   Stdev(ms)    Rate(M/s)   Per Row(ns)   Relative
```

Comment:
I think we don't need benchmark for all tables, do you have specific reason for that?

Reply:
> I think we don't need benchmark for all tables, do you have specific reason for that?

This doesn't seem to be needed, they seem to generate fast, I will remove it.